src/index.html

```diff
@@ -35,21 +40,19 @@ <h2>Recent Scores</h2>
 
 				<section class="add-section">
 					<div>
-						<h2>Add your Score</h2>
+						<h2>Add Your Score</h2>
 					</div>
 					<form action="" class="add-form" id="add-form">
-						<input type="text" id="name-field" placeholder="Your name"/>
-						<input type="text" id="score-field" placeholder="Your score"/>
-						<input type="submit" class="button"></input>
+						<input type="text" required id="name-field" placeholder="Your name"/>
+						<input type="number" required id="score-field" placeholder="Your score"/>
+						<button value="Submit Score" class="button-89">Submit Score</button>
 					</form>
```

Comment:
- Kindle make sure the impute fields and the submit score  button have the appropriate design as it shows in the design provided, the color of the border, the disposition, size and the format it will make your design much more to the design provided as was required in the project :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/204088566-eaba8ed2-a5cb-472b-aff1-13c24a87d030.png)

## Design provides

![image](https://user-images.githubusercontent.com/95297251/204088587-e5788f51-1cd8-4880-9f4d-2ac070abaff9.png)